libipl/p10/ipl0.C

```diff
@@ -417,15 +419,21 @@ static int initialize_and_check_clock_chip(uint8_t& clock_count)
 			continue;
 		}
 
+		clk_pos = pdbg_target_index(clock_target);
```

Comment:
Why you are using an `index` instead of a `POSITION` attribute? I think you are requested to include the POSITION attribute right?

Reply:
as per my understanding, you have changed it to index attribute

Reply:
No, POSITION is added and index is common property for all targets, please use the right based on your needs

Reply:
Done